sdk/keyvault/microsoft-azure-keyvault-webkey/src/test/java/com/microsoft/azure/keyvault/webkey/test/EcValidationTests.java

```diff
@@ -114,4 +118,31 @@ private static void signVerify(PublicKey publicKey, PrivateKey privateKey, JsonW
             .put(JsonWebKeyCurveName.P_521, "SHA512withECDSA")
             .put(JsonWebKeyCurveName.P_256K, "NONEwithECDSA")
             .build();
+
+    private static int getJavaVersion() {
+        // java.version format:
+        // 8 and lower: 1.7, 1.8.0
```

Comment:
There's also underscores to worry about

Java 8 or lower: 1.6.0_23, 1.7.0, 1.7.0_80, 1.8.0_211
Java 9 or higher: 9.0.1, 11.0.4, 12, 12.0.1